app/controllers/good_deeds_controller.rb

```diff
@@ -3,7 +3,12 @@ def index
     @deeds = GoodDeedFacade.new
   end
 
-  def login
-    
+  def new
+    return unless current_user.nil?
+
+    redirect_to login_path
+    flash[:alert] = "You must be logged in to create a new good deed"
   end
-end
\ No newline at end of file
+
+  def login; end
```

Comment:
Moving forward, let's move the login and logout functionality to a sessions controller.